coffeelint.json

```diff
@@ -0,0 +1,19 @@
+{
+    "check_scope": {
+      "module": "coffeescope2",
+      "level": "warn",
+      "environments": [],
+      "globals": {},
+      "overwrite": false,
+      "same_scope": false,
+      "shadow": false,
+      "shadow_builtins": false,
+      "shadow_exceptions": [],
+      "undefined": false,
+      "hoist_local": true,
+      "hoist_parent": true,
+      "unused_variables": true,
+      "unused_arguments": false,
+      "unused_classes": false
```

Comment:
I'm not sure, it'd be nice to disable only this specific check in only this specific repo, rather than checking them all.  At the same time I think it would be ok to disable generally, it's just I don't think it's actually needed anywhere else (although I'm not certain).